### ISSUE TITLE
Bug with chalk (take browser env instead of node)

### ISSUE THREAD

**Issue by rharkor** [privatenumber/pkgroll#83]
### Problem

I am using chalk as a dev dependency in my project but when I build it it will take the browser version of chalk instead of the node one. 
So when I run my program it will throw "navigator is not defined"

### Expected behavior

Take the node.js version of chalk

### Minimal reproduction URL

https://github.com/rharkor/repro-pkgroll-chalk-bug

### Version

2.4.2

### Node.js version

20.10.0

### Package manager

npm

### Operating system

Linux

### Bugs are expected to be fixed by those affected by it

- [ ] I'm interested in working on this issue

### Compensating engineering work financially will speed up resolution

- [ ] I'm willing to offer financial support

**Comment by rharkor** [comment]
It appears that it come from esbuild that make choose the "browser" platform by default, with their cli we can choose the platform and use node but I don't know how to do it with pkgroll..

Rf: https://esbuild.github.io/api/#platform

**Comment by rharkor** [comment]
I finally found it by searching in the code (btw there's a small part about it in the doc, don't know why I didn't find it earlier).

For those having the same problem use the following option:
`--export-condition=node`

I also found that it may be renamed in the future to `conditions` so maybe use both so you don't have to come back to it later

TLDR;
`pkgroll --export-condition=node -conditions=node`
